python/res/fm/ecl/ecl_run.py

```diff
@@ -173,33 +175,39 @@ def initMPI(self):
 
 
     def execSerialEclipse(self):
-        process = subprocess.Popen(
-            [
-                self.sim.executable,
-                self.base_name
-            ],
-            env=self.sim.env
-        )
-        process.communicate()
-        return process.returncode
+        log_name = "{}.LOG".format(self.base_name)
+
+        with open(log_name, 'wb') as log_file:
+            process = subprocess.Popen(
+                [
+                    self.sim.executable,
+                    self.base_name
+                ],
+                env=self.sim.env,
+                stdout=subprocess.PIPE,
+            )
+            return self._await_process_tee(process, sys.stdout, log_file)
 
 
     def execParallellEclipse(self):
+        log_name = "{}.LOG".format(self.base_name)
```

Comment:
Should we unify this logic to ensure that parallell and serial is consistent?

Reply:
Unified into `execEclipse`: ef3106eeb494eb120d7dfb86d5e3ca23813cf15e